force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1,5 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomLabels xmlns="http://soap.sforce.com/2006/04/metadata">
+    <labels>
+        <fullName>accordionSection_ToggleInstructionsWhenOpen</fullName>
+        <categories>accordionSection</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Click to toggle the section. This section is open.</shortDescription>
```

Comment:
Did we have these reviewed by docs yet?

Reply:
I copied it directly from caseman. I can tag Lauren so she can look at it